### PR TITLE
Clean Up Mobile Dropdown Overlay on window resize

### PR DESCRIPTION
**Pull request by will-nemo**

### CODE REVIEW

File: src/components/Toolbar/Toolbar.jsx [hunk 1]
```diff
@@ -15,14 +15,47 @@ function Toolbar() {
...
+  }, [isOpen]);
+
+  /**
+   * Callback used by resize eventListener to close mobile nav bar in desktop view
+   * @returns {void}
+   * @callback
+   */ 
+  const memoNavCleanUp = useCallback(() => {
+    //if mobile bar is open and screen width is greater than medium breakpoint (see _screens.scss)
+    return (isOpen && window.innerWidth > 960) ? handleToggle() : null;
```
Comment (ambid17) [inline src/components/Toolbar/Toolbar.jsx:39]: const for the breakpoint

Comment (will-nemo) [reply src/components/Toolbar/Toolbar.jsx:39]: good call

File: src/components/Toolbar/Toolbar.jsx [hunk 2]
```diff
@@ -15,14 +20,47 @@ function Toolbar() {
...
+      window.removeEventListener('resize', memoNavCleanUp);
+    };
+  }, [memoNavCleanUp]);
+
+  /************************************
+   * Functions
+   ************************************/
 
+  /**
+   * Applys open rule to className to add open attributes to parts of navbar
```
Comment (Fabricevladimir) [inline src/components/Toolbar/Toolbar.jsx:61]: VS Code caught a spelling error. Applys should be Applies.

Comment (will-nemo) [reply src/components/Toolbar/Toolbar.jsx:61]: :,(

File: src/components/Toolbar/Toolbar.jsx [hunk 3]
```diff
@@ -15,14 +20,47 @@ function Toolbar() {
...
+   * Callback used by resize eventListener to close mobile nav bar in desktop view
+   * @returns {void}
+   * @callback
+   */ 
+  const memoNavCleanUp = useCallback(() => {
+    //if mobile bar is open and screen width is greater than medium breakpoint (see _screens.scss)
+    return (isOpen && window.innerWidth > MEDIUM_BREAKPOINT) ? handleToggle() : null;
+  }, [isOpen, handleToggle]);
+
+  useEffect(() =>{
```
Comment (0xLeif) [inline src/components/Toolbar/Toolbar.jsx:47]: Consistent Spacing: `useEffect(() => {`

File: src/components/Toolbar/__tests__/Toolbar.test.jsx [hunk 4]
```diff
@@ -55,4 +56,22 @@ describe('Toolbar Component Tests', function() {
     expect(queryByText('Partners')).toBeInTheDocument();
     expect(queryByText('Sign Up')).not.toBeInTheDocument();
   });
+
+  test('Toolbar resized to width over medium screen breakpoint; Mobile Navigation Menu Closed', () => {
+    const { queryByLabelText, getByTestId } = setup();
+    window.innerWidth = MEDIUM_BREAKPOINT - 10;
+    window.dispatchEvent(new Event('resize'));
+    queryByLabelText('hamburger')
```
Comment (b1995) [inline src/components/Toolbar/__tests__/Toolbar.test.jsx:64]: I am confused to why this line is here, I don't think it does anything.

Comment (will-nemo) [reply src/components/Toolbar/__tests__/Toolbar.test.jsx:64]: good catch, shall remove

**Pull request merged**